Gemfile

```diff
@@ -4,7 +4,7 @@ group :test do
   gem "rb-inotify", '< 0.10.0' if RUBY_VERSION < '2.2.0'
   gem "public_suffix", "~> 3.0.0"
   gem "listen", "~> 3.0.0"
-  gem "puppet", ENV['PUPPET_VERSION'] || '~> 4.2.0'
+  gem "puppet", ENV['PUPPET_VERSION'] || '~> 4.6.2'
```

Comment:
Shouldn't `~> 4.6.0` basically give you the upper bound on `4.6.x` ?

Reply:
`'~> 4.6.2'` == `'4.6.2 < version < 4.7.0'`
`'~> 4.6.0'` == `'4.6.0 < version < 4.7.0'`

I didn't assume it works on 4.6.0 because on Travis we test with 4.6.2.